app/views/questions/upload-form.html

```diff
@@ -1,30 +1,27 @@
 {% set title = data['test-type'] %}
-{% set warning %}
-If your {{ title }} form is not complete, then it may cause problems with your application.
-{% endset %}
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    {{ govukInsetText({
-      text: warning
-    }) }}
     <form class="form" action="upload-supporting-documentation">
       <div class="govuk-caption-m">{{ title }}</div>
       <h1 class='govuk-heading-l'>Upload application form</h1>
       <p class="govuk-body">
-        Upload your completed {{ title }} form below.
+        Upload the official document you have completed to apply for a {{ title }}.
       </p>
       {{ govukFileUpload({
       id: "application-upload",
       name: "application-upload",
       label: {
-      html: "<strong>Upload application form</strong>"
+      html: "<strong>Choose an application form to upload</strong>"
```

Comment:
@Ella-Welsby I've replaced this HTML with the Nunjucks equivalent again, just because it makes the hint text easier to associate with the input